microarray/src/org/labkey/microarray/query/MicroarrayUserSchema.java

```diff
@@ -155,10 +153,6 @@ public TableInfo getAnnotationSetTable()
         return getTable(TABLE_FEATURE_ANNOTATION_SET);
     }
 
-    public TableInfo getAnnotationTable()

```

Comment:
I think we still need the FeatureAnnotation table.

Reply:
As an addon to my comment above, if you think it'd be safer to keep it without finding the missing use case, I'd be happy to do that

Reply:
This method isn't used in module currently. Existing use cases are using getTable() which calls createTable() instead of going through this wrapper method. I think it's safe and good to delete it, especially since it calls the deprecated version of getTable() at the moment.

Reply:
Cool, sounds like this is good to go then.